src/algorithms/digi/PhotoMultiplierHitDigiConfig.h

```diff
@@ -7,19 +10,20 @@ namespace eicrecon {
     public:
 
       // random number generator seed
-      unsigned long seed = 0;
+      // FIXME: don't use 0 if `TRandomMixMax` is the RNG, it can get "stuck"
+      unsigned long seed = 0; // seed for RNG (note: `0` might mean "unique" seed)
 
       // triggering
-      double hitTimeWindow = 20.0;   // [ns]
-      double timeStep      = 0.0625; // [ns]
-      double speMean       = 80.0;
-      double speError      = 16.0;
-      double pedMean       = 200.0;
-      double pedError      = 3.0;
+      double hitTimeWindow = 20.0;  // time gate in which 2 input hits will be grouped to 1 output hit // [ns]
+      double timeStep      = 16.0;  // number of TDC counts per unit time // [ns^-1]
+      double speMean       = 80.0;  // mean ADC counts for a single photon
+      double speError      = 16.0;  // sigma of ADC counts for a single photon
+      double pedMean       = 200.0; // mean ADC counts for the pedestal
+      double pedError      = 3.0;   // sigma of ADC counts for the pedestal
 
       // SiPM pixels
       bool   enablePixelGaps = false; // enable/disable removal of hits in gaps between pixels
-      double pixelSize       = 3.0;   // [mm] // pixel (active) size
+      double pixelSize       = 3.0;   // pixel (active) size // [mm]
```

Comment:
Multiply with units, e.g.
```suggestion
      double pixelSize       = 3.0 * dd4hep::mm;   // pixel (active) size
```

Reply:
I thought we agreed to convert numbers to a standard system [mm,GeV,ns,radians] as soon as possible, so putting a configuration parameter in the DD4hep unit system is not necessary; instead any numbers read from the DD4hep service should also be converted to the standard system by division. Are we changing the standard again?

Reply:
Yes, but we put them into this standard unit system by entering them with those units when we provide a numerical value. Starting from then on they are in our unit system. We should not just enter their value without a unit, because then it is not obvious which units they are in. See also EICrecon factory parameters.

Reply:
So are we now re-allowing config params to be in DD4hep units? Should this be enforced as a standard? That is fine with me, but should be done in a separate PR.
In that case, https://github.com/eic/EICrecon/pull/390 could be useful.

Reply:
(when/if we move to some config param `yaml` file or something)

Reply:
We do want units on parameters, and they should be dd4hep units. That's what all algorithms and factories are doing, e.g. https://github.com/eic/EICrecon/blob/main/src/algorithms/calorimetry/CalorimeterHitDigi.h#L47 and https://github.com/eic/EICrecon/blob/main/src/detectors/BEMC/CalorimeterHit_factory_EcalBarrelImagingRecHits.h#L37. Only time doesn't always have units because dd4hep doesn't have a consistent unit for times.